### PR TITLE
Add support for C++ extensions

### PR DESCRIPTION
**Pull request by ecourtois**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -122,22 +122,32 @@ To enable pre-commit hooks prevent committing code not respecting linting and fo
...
+
+A Docker Compose service is provided to build a binary distribution package for linux/arm64 platform.
 
 ```bash
-uv build --sdist
+docker compose up --build build_wheel
 ```
 
-This will produce `dist/cogip_tools-1.0.0.tar.gz`.
+This will produce `dist/cogip_tools-1.0.0-cp312-cp312-linux_aarch64.whl`.
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply docs/install.md:135]: The filename is generated by the build backend. It respects PEP491: https://peps.python.org/pep-0491/#file-format.
The first `cp312` indicates the CPython version compatibility. Here it refers to CPython 3.12.
The second `cp312` is the ABI tag, which tells the specific Python implementation and version for compatibility. Here the wheel is compatible only with CPython 3.12.

File: docs/install.md [hunk 2]
```diff
@@ -122,22 +122,32 @@ To enable pre-commit hooks prevent committing code not respecting linting and fo
...
 
 This package can be copied to the Raspberry Pi and installed to deploy the Python tools:
 
 ```bash
-uv tool install cogip_tools-1.0.0.tar.gz
+uv pip install cogip_tools-1.0.0-cp312-cp312-linux_aarch64.whl
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply docs/install.md:140]: Same answer as before.

File: pyproject.toml [hunk 3]
```diff
@@ -90,29 +90,27 @@ explicit = true
...
-    "static/js/external/*",
-    "templates/*"
-]
-"cogip.tools.dashboard_beacon" = [
-    "static/css/*",
-    "static/css/purged/*",
-    "static/img/*",
-    "static/js/*",
-    "static/js/external/*",
-    "templates/*"
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply pyproject.toml:112]: Actually I did not _remove_ the static files.
The new build backend is including all files not excluded (see `sdist.exclude`), where the former backend was including the specified files only.

**Review by mlecriva — APPROVED** [review]
LGTM

**Pull request merged**